salt/cloud/clouds/digitalocean.py

```diff
@@ -298,7 +298,8 @@ def create(vm_):
         'size': get_size(vm_),
         'image': get_image(vm_),
         'region': get_location(vm_),
-        'ssh_keys': []
+        'ssh_keys': [],
+        'tags': []
```

Comment:
Below, it looks like this is expected to be a string? IMHO, we should either actually make it a list and specify it that way in the configuration file, which I think is preferable, or we should set this to default to an empty string.

Reply:
yea the way i write it was just based on the `ssh_keys` param which does something similar.

```
    if ssh_key_names:
        for key in ssh_key_names.split(','):
            kwargs['ssh_keys'].append(get_keyid(key))

    key_filename = config.get_cloud_config_value(
        'ssh_key_file', vm_, __opts__, search_global=False, default=None
    )
```

We send it to DO as an array but read it from the config as a string. i can look into changing everything to an array.

Reply:
Thanks, @kkoch986

Reply:
Any update here @kkoch986

Reply:
just updated it sorry for the delay